index.html

```diff
@@ -318,41 +318,30 @@ <h3 id="unacceptablebehavior">
           <li>Use of coded language (also known as "dog whistles") used to
           rally support for hate groups or to intimidate vulnerable groups.
           </li>
+          <li><a>Patronizing</a> language or behavior:
+            <ul>
+              <li>Intentionally or unintentionally making assumptions about the skills or knowledge of others, such as using language that implies the audience is uninformed on a topic (e.g. interjections like "I can't believe you don't know about [topic]").
+              </li>
+              <li>Assuming that particular groups of people are technically unskilled due to their characteristics (e.g., “So easy your grandmother could do it”, which implies an older woman might not be technically competent).
```

Comment:
This is more an example of prejudice than of patronizing behavior.  And it is still phrased in an overly general way, because "characteristics" that are relevant to one's technical skill _should_ be considered, while those that are irrelevant to one's technical skill should not.  I suggest deleting this bullet from this location, and perhaps instead using the "grandmother" example elsewhere in the document as an example of prejudice.